noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -71,6 +71,17 @@ impl BaseRollupInputs {
             == self.constants.global_variables.version, "kernel version does not match the rollup version"
         );
 
+        let rollup_validation_requests = self.kernel_data.public_inputs.rollup_validation_requests;
+
+        // Verify the max block number
+        // TODO: why is block_number a Field and not u32?
```

Comment:
Is this resolved or you need a hand or? 👀

Reply:
Created https://github.com/AztecProtocol/aztec-packages/issues/5345, this seems non-trivial but also not urgent and not worth blocking this PR.